api/src/endpoints/addCommentEndpoint.tsx

```diff
@@ -0,0 +1,182 @@
+import express from 'express';
+import { Comment, Info, User } from 'entities';
+import { Gateways, err } from 'gateways';
+import { getAuthenticatedUserId } from '../parseAuth0User';
+import { authenticationRequiredError } from 'gateways/src/errors';
+import { sesClient } from '../libs/sesClient';
+import { formatTimestamp } from '../../../app/src/accessors/formatTimestamp';
+import { SendEmailCommand } from '@aws-sdk/client-sesv2';
+import { renderToString } from 'react-dom/server';
+import { CommentBody } from '../../../components/src/components/Comments/CommentBody'
+import React from 'react';
+import Markdown from 'react-markdown';
+
+export const addCommentEndpoint = ({
+  app,
+  gateways,
+}: {
+  app: any;
+  gateways: Gateways;
+}) => {
+  const { saveComment, getUser, getInfo } = gateways;
+
+  let sendCommentEmail = async (
```

Comment:
stylistic preference

```suggestion
  const sendCommentEmail = async (
```